core/services/ocr2/plugins/llo/helpers_test.go

```diff
@@ -114,7 +114,7 @@ func startMercuryServer(t *testing.T, srv *mercuryServer, pubKeys []ed25519.Publ
 		t.Fatalf("[MAIN] failed to listen: %v", err)
 	}
 	serverURL = lis.Addr().String()
-	s := wsrpc.NewServer(wsrpc.Creds(srv.privKey, pubKeys))
+	s := wsrpc.NewServer(wsrpc.WithCreds(srv.privKey, pubKeys))
```

Comment:
The Mercury Server has been updated already (https://github.com/search?q=repo%3Asmartcontractkit%2Fmercury-pipeline+wsrpc.Creds&type=code)